etc/grafana/dashboards/overview.json

```diff
@@ -1,14 +1,4 @@
 {
-  "__inputs": [
-    {
-      "name": "DS_PROMETHEUS",
-      "label": "Prometheus",
-      "description": "",
-      "type": "datasource",
-      "pluginId": "prometheus",
-      "pluginName": "Prometheus"
-    }
-  ],
```

Comment:
Please export the dashboard for external use using Grafana, without this information people will have to manually adjust the dashboard if their source is not named Prometheus

Reply:
made it work for both importing into people's own Grafana instances and using it inside pre-configured docker compose